### PR TITLE
ENH:Added single and bulk event authentication based on work I did wi…

### PR DESCRIPTION
**Pull request by arkilic**
…th @klauer

### CODE REVIEW

File: metadataservice/schema/bulk_events.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                "type": "object",
+
+                "properties": {
+                    "data": {
+                        "type": "object",
+                        "description": "The actual measument data"
+                    },
+                    "timestamps": {
+                        "type": "object",
+                        "description": "The timestamps of the individual measument data"
```
Comment (klauer) [inline metadataservice/schema/bulk_events.json]: Typo here `measument`

File: metadataservice/schema/bulk_events.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    "patternProperties": {
+        "^.*$": {
+            "type": "array",
+            "items": {
+                "type": "object",
+
+                "properties": {
+                    "data": {
+                        "type": "object",
+                        "description": "The actual measument data"
```
Comment (klauer) [inline metadataservice/schema/bulk_events.json]: Typo here `measument`

File: metadataservice/server/utils.py [hunk 3]
```diff
@@ -11,6 +11,22 @@
 __all__ = ['_verify_handler', '_unpack_params',
            '_return2client', '_stringify_data']
 
+
+
+
```
Comment (klauer) [inline metadataservice/server/utils.py]: So many blank lines. Wasting precious bytes.

Comment (arkilic) [inline metadataservice/server/utils.py]: I got rid off it :100:

**Pull request merged**